src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/SoulboundListener.java

```diff
@@ -62,7 +62,7 @@ public void onDamage(PlayerDeathEvent e) {
         while (drops.hasNext()) {
             ItemStack item = drops.next();
 
-            if (SlimefunUtils.isSoulbound(item)) {
+            if (SlimefunUtils.isSoulbound(item) && Slimefun.isEnabled(p, item, false)) {
```

Comment:
Doing this all twice leaves a bit of a redundant aftertaste, especially because `isSoulbound()` and `isEnabled()` both do `SlimefunItem.getByItem()` checks :/